code/frameworks/angular/src/client/docs/__testfixtures__/doc-button/input.ts

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
 // @ts-nocheck
```

Comment:
looks like `ts-nocheck` is actually valid here because of the decorators, so I've allowed it.